indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -508,6 +504,7 @@ public void run()
             if (isEndSequenceOffsetsExclusive() &&
                 createSequenceNumber(record.getSequenceNumber()).compareTo(
                     createSequenceNumber(endOffsets.get(record.getPartitionId()))) >= 0) {
+              stillReading = false;
```

Comment:
I guess this is the change to fix the bug of stuck on resume. But, it looks to be better to fix the [assignPartitions](https://github.com/apache/incubator-druid/blob/master/indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java#L1033-L1038) method which is the method to check how many offsets are remaining per partition. This method is also called before starting the read loop.

Reply:
Yes, thanks, will update :+1: